packages/block-library/src/heading/block.json

```diff
@@ -1,5 +1,5 @@
 {
-	"apiVersion": 2,
+	"apiVersion": 1,
```

Comment:
This change has to be reverted after testing.

Reply:
Are we not using useBlockProps here?

Reply:
I added it only to make the testing simpler and ensure it doesn't fail e2e tests that use production files.